crates/derive/src/attributes/mod.rs

```diff
@@ -703,7 +703,7 @@ mod tests {
             )),
             eip_1559_params: None,
         };
+        assert_eq!(payload.transactions.as_ref().unwrap().len(), 10);
         assert_eq!(payload, expected);
-        assert_eq!(payload.transactions.unwrap().len(), 4);
```

Comment:
This feels odd - something probably went wrong here... how did we suddenly get 6 new transactions?

Reply:
I'm not sure tbh, feels like a config with revm?

Reply:
We've debugged offline - this is because of https://github.com/alloy-rs/op-alloy/pull/170. If fjord is active, ecotone must be active so this should be 10 txs. The test was really originally invalid so this updates the behavior.